File: 3872-find-most-frequent-vowel-and-consonant/3872-find-most-frequent-vowel-and-consonant.py
import collections

class Solution:
    def maxFreqSum(self, s: str) -> int:
        counted = collections.Counter(s)
        vowel = set(('a', 'e', 'i', 'o', 'u'))
        answer = 0
        v_max = 0
        c_max = 0
        for k, v in counted.items():
            if k in vowel:
                v_max = max(v_max, v)
            else:
                c_max = max(c_max, v)
        return v_max + c_max

File: 3872-find-most-frequent-vowel-and-consonant/3872-find-most-frequent-vowel-and-consonant.go
func maxFreqSum(s string) int {
    v_max := 0
    c_max := 0
    vowels := map[byte]bool{
        'a': true, 
        'e': true,
        'i': true,
        'o': true,
        'u':true,
    }
    counter := make(map[byte]int)
    for i:=0 ;i<len(s);i++{
        counter[s[i]]++
    }
    for k,v := range counter {
        if vowels[k] {
            v_max = max(v, v_max)
        } else {
            c_max = max(v, c_max)
        }
    }
    return c_max + v_max
}
